docs/standard/native-interop.md

```diff
@@ -326,6 +326,296 @@ public class StatClass {
 
 The `StatClass` class represents a structure that is returned by the `stat` system call on UNIX systems. It represents information about a given file. The class above is the stat struct representation in managed code. Again, the fields in the class have to be in the same order as the native struct (you can find these by perusing man pages on your favorite UNIX implementation) and they have to be of the same underlying type.
 
+#### Customizing Struct Marshalling
+
+Sometimes the default marshalling rules for fields in a structure aren't exactly what you need. We provide a few extension points for you to customize how your struct is marshalled to native code.
```

Comment:
"We provide a few extension points for you to customize how your struct is marshalled to native code." => "Extension points are provided to customize how structs are marshalled to native code."

Reply:
In general technical documents should avoid using pronouns. Reference the action or scenario.